telegram/chatmember.py

```diff
@@ -456,6 +475,12 @@ class ChatMemberAdministrator(ChatMember):
             administrator can delete messages of other users.
         can_manage_voice_chats (:obj:`bool`, optional): :obj:`True`, if the
             administrator can manage voice chats.
+
+            .. deprecated:: 13.12
```

Comment:
"Contains the same value as *_voice_* for backwards compatibility" or similar wording. same everywhere, where applicable